lib/spack/spack/build_systems/python.py

```diff
@@ -210,7 +210,13 @@ def install(self, spec, prefix):
 
     def install_args(self, spec, prefix):
         """Arguments to pass to install."""
-        args = ['--prefix={0}'.format(prefix)]
+        python_version = self.spec['python'].version.up_to(2)
+        python_string = 'python{0}'.format(python_version)
+        path_to_install_lib = os.path.join(
+            prefix, 'lib', python_string, 'site-packages')
+
+        args = ['--prefix={0}'.format(prefix),
+                '--install-lib={0}'.format(path_to_install_lib)]
```

Comment:
minor : could you add reference to Spack PR/issue where this was suggested? If you add it to commit message, we will have reference.

Reply:
it was already done!